miniquad-derive/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "miniquad-derive"
+version = "0.1.0"
+edition = "2018"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+proc-macro2 = "1.0.26"
+# need `extra-traits` feature for `Attribute` to implement `PartialEq`
+syn = { version = "1.0.70", features = ["derive", "parsing", "printing", "clone-impls", "proc-macro", "extra-traits"] }
```

Comment:
Using `syn` will triple miniquad's compilation time. I usually try to use proc-macro dependencies that do not use `syn`, like nanoserde. 

Maybe we can move this derive macro a third-party crate instead being a part of miniquad?

Reply:
> Using syn will triple miniquad's compilation time.
That's too bad!